src/stores/risk.store.js

```diff
@@ -61,34 +69,40 @@ class RiskStore {
     .filter(o=> o)
   }
 
-  init = async ()=> {
-    if(true) {
-      const data = await mainStore['risk_params_request'] 
-      this.utilization = await this.getUtilization()
+    init = async () => {
+        if (true) {
+            
+            const { computeReverseSandbox } = window.APP_CONFIG.feature_flags
```

Comment:
this is the new feature flag 'computeReverseSandbox' that must be set to true to compute  the reverse solver values